cli/src/installers/drizzle.ts

```diff
@@ -42,10 +61,85 @@ export const drizzleInstaller: Installer = ({
     "project1_${name}",
     `${scopedAppName}_\${name}`
   );
+
   let configContent = fs.readFileSync(configFile, "utf-8");
+  const dbType = (
+    {
+      postgres: "pg",
+      neon: "pg",
+      sqlite: "sqlite",
+      mysql: "mysql",
+      planetscale: "mysql",
+    } as const
+  )[databaseProvider];
   configContent = configContent.replace("project1_*", `${scopedAppName}_*`);
+  if (databaseProvider !== "mysql" && databaseProvider !== "planetscale") {
+    configContent = configContent.replace(
+      "mysql2",
+      {
+        postgres: "pg",
+        neon: "pg",
+        sqlite: "better-sqlite",
+      }[databaseProvider]
+    );
+    if (databaseProvider === "sqlite")
+      configContent = configContent.replace("connectionString:", "url:");
+
+    schemaContent = schemaContent.replace(
+      "drizzle-orm/mysql-core",
+      `drizzle-orm/${dbType}-core`
+    );
+    schemaContent = schemaContent.replaceAll(
+      "mysqlTableCreator",
+      `${dbType}TableCreator`
+    );
+    schemaContent = schemaContent.replaceAll(".onUpdateNow()", "");
+    if (dbType === "sqlite") {
+      if (packages?.nextAuth.inUse) {
+        schemaContent = schemaContent.replace("  varchar,\n", "");
+        schemaContent = schemaContent.replace("  bigint,\n", "");
+      }
+      schemaContent = schemaContent.replace("  timestamp,\n", "");
+      schemaContent = schemaContent.replaceAll("varchar", "text");
+      schemaContent = schemaContent.replaceAll("bigint", "int");
+      schemaContent = schemaContent.replace(
+        /timestamp\("([a-zA-Z\-_]+)", { mode: "date" }\)/g,
+        'int("$1", { mode: "timestamp_ms" })'
+      );
+      schemaContent = schemaContent.replace(
+        /timestamp\("([a-zA-Z\-_]+)"\)/g,
+        'int("$1", { mode: "timestamp" })'
+      );
+      schemaContent = schemaContent.replace(
+        `timestamp("emailVerified", {
+    mode: "date",
+    fsp: 3,
+  })`,
+        'int("emailVerified", { mode: "timestamp" })'
+      );
+      schemaContent = schemaContent.replaceAll(
+        ".primaryKey().autoincrement()",
+        ".primaryKey({ autoIncrement: true })"
+      );
+    }
+    if (dbType === "pg") {
+      schemaContent = schemaContent.replace("  bigint,\n", "");
+      schemaContent = schemaContent.replace(
+        "  int,\n",
+        "  integer as int,\n  serial,\n"
+      );
+      schemaContent = schemaContent.replace(
+        'id: bigint("id", { mode: "number" }).primaryKey().autoincrement(),',
+        'id: serial("post_id").primaryKey(),'
+      );
+      schemaContent = schemaContent.replace("fsp: ", "precision: ");
```

Comment:
all this looks prone to error at the first change we make. just having them as separate files will be easier to maintain